Formula/kent-tools.rb

```diff
@@ -32,7 +32,7 @@ def install
     args << "PNGLIB=-L#{libpng.opt_lib} -lpng -lz"
     args << "PNGINCL=-I#{libpng.opt_include}"
     args << "MYSQLINC=#{mysql.opt_include}/mysql"
-    args << "MYSQLLIBS=-lmysqlclient -lz"
+    args << "MYSQLLIBS=-L#{mysql.opt_lib} -lmysqlclient -lz -lstdc++"
```

Comment:
Just so we have a record of why this was added, what is `-lstdc++` needed for?

Reply:
I guess some tools accidentally contain C++ source, and only `CC` (C compiler/linker) is used throughout `Makefile`s in the source tree. Usually `-lstdc++`/`-lc++` should not be added manually because `CXX` (C++ compiler/linker) handles it.